packages/valory/skills/price_estimation_abci/behaviours.py

```diff
@@ -52,11 +52,11 @@
 )
 
 
-# This safeTxGas value is calculated from experimental values plus
-# a 10% buffer and rounded up. The Gnosis safe default value is 0 (max gas)
+# The Gnosis safe default value is 0 (max gas)
 # https://help.gnosis-safe.io/en/articles/4738445-advanced-transaction-parameters
+# This value allows for straightforward gas estimation.
 # More on gas estimation: https://help.gnosis-safe.io/en/articles/4933491-gas-estimation
-SAFE_TX_GAS = 120000
+SAFE_TX_GAS = 0
```

Comment:
At some point we can use exact values, but it can interfere with gas estimation and needs to be consistently applied throughout.